### ISSUE TITLE
Native Android lib v1.48.0 cause build error

### ISSUE THREAD

**Issue by nghuuhieu1994** [didomi/unity#24]
Revert android lib to `v1.47.0` fix the issue
This can be reproduce with android lib version greater or equal `v1.47.1`

Build Error Log

> java.lang.UnsupportedOperationException: This feature requires ASM7
	at org.objectweb.asm.ClassVisitor.visitNestHost(ClassVisitor.java:150)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:541)
	at org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
	at com.android.builder.desugaring.DesugaringClassAnalyzer.analyze(DesugaringClassAnalyzer.java:153)
	at com.android.builder.desugaring.DesugaringClassAnalyzer.analyzeJar(DesugaringClassAnalyzer.java:100)
	at com.android.builder.desugaring.DesugaringClassAnalyzer.analyze(DesugaringClassAnalyzer.java:68)
	at com.android.build.gradle.internal.tasks.DesugarIncrementalHelper.lambda$getIncrementalData$5(DesugarIncrementalHelper.java:176)
	at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1424)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
FAILURE: Build failed with an exception.
What went wrong:
Execution failed for task ':launcher:dexBuilderRelease'.
> java.lang.UnsupportedOperationException: This feature requires ASM7

**Comment by pmerlet-at-didomi** [comment]
Hello, thanks for the info.

If you use several libraries in your project, maybe adding multidex could solve this issue: https://forum.unity.com/threads/enable-multidex-support-for-unity-2020.990440/#post-7014889 , can you try it?

**Comment by pmerlet-at-didomi** [comment]
I also encountered the issue when using gradle 6.8, apparently it is a known issue in this version. It can be fixed by using gradle 6.1.1 (the default version used by Unity), or update to a more recent one, for example 7.0.0.

`gradle-wrapper.properties`:
```
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists

```
